config/karma/shared.karma.conf.js

```diff
@@ -193,6 +196,9 @@ function getConfig(config) {
       level: 'log'
     },
     reporters: ['mocha', 'coverage'],
+    mochaReporter: {
+      ignoreSkipped: true
```

Comment:
We can remove this if you don't like it. It was getting annoying seeing all the skipped tests printed on the screen when using `fit` or `fdescribe`.

Reply:
I think until we go to Jasmine 3, I'd want to leave this.